README.rst

```diff
@@ -428,6 +428,48 @@ TMT_REBOOT_COUNT
     for requesting the reboot, variables ``REBOOTCOUNT`` and
     ``RSTRNT_REBOOTCOUNT`` contain number of reboots as well.
 
+Plugin options
+--------------
+
+Each plugin option can be also specified via environment variable.
+Variables follow a naming scheme utilizing plugin name, step it
+belongs to, and the option name:
+
+    ``TMT_PLUGIN_${STEP}_${PLUGIN}_${OPTION}``
+
+All values are upper-cased, with dashes (``-``) replaced by
+underscores (``_``).
+
+For example, an execute plugin "tmt" would run with verbosity
+equal to ``-vvv``::
+
+    TMT_PLUGIN_EXECUTE_TMT_VERBOSE=3 tmt run ... execute -h tmt ...
+
+Command-line takes precedence over environment variables, therefore
+``-v`` would undo the effect of environment variable, and reduce
+verbosity to one level only::
+
+    TMT_PLUGIN_EXECUTE_TMT_VERBOSE=3 tmt run ... execute -h tmt -v ...
+
+Environment variables - just like command-line options - take
+precedence over values stored in files. For example, consider the
+following discover step::
+
+    discover:
+        how: fmf
+        url: https://example.org/
+
+The following commands would override the URL::
+
+    tmt run ... discover -h fmf --url https://actual.org/ ...
+
+    TMT_PLUGIN_DISCOVER_FMF_URL=https://actual.org/ tmt run ...
+
+For setting flag-like option, 0 and 1 are the expected value. For
+example, an interactive mode would be enabled in this run::
+
+    TMT_PLUGIN_EXECUTE_TMT_INTERACTIVE=1 tmt run ... execute -h tmt ...
+
```

Comment:
here at the and I would note the current limitation in case a default plugin is used or something ...

Reply:
Fixed in e88cadc2